Dockerfile

```diff
@@ -1,107 +1,29 @@
-FROM openjdk:10
-
-ENV CLOJURE_VERSION=1.9.0.381
-
-WORKDIR /tmp
-
-RUN wget https://download.clojure.org/install/linux-install-$CLOJURE_VERSION.sh \
-    && chmod +x linux-install-$CLOJURE_VERSION.sh \
-    && ./linux-install-$CLOJURE_VERSION.sh
-
-RUN clojure -e "(clojure-version)"
-
-WORKDIR /opt/urbest
-
-# install cypress dependencies, aws and jq
```

Comment:
I just removed it, because with the approach we are going to use we don't need cypress in a monolith container and we'll use the cypress provided one https://github.com/urbestteam/z-frontend/pull/1460/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R342